### ISSUE TITLE
Visualize a simulation run from a log file

### ISSUE THREAD

**Issue by mickvangelderen** [oliverlee/phobos#167]
In order to analyze simulations and to capture high quality simulations, we want to be able to replay a simulation. By recording the serial communications to a file exactly we can achieve this. 

There are some differences between live and replay visualisation. During live visualisation, we want to know the latest state and render it. 

When replaying, all states matter. We want to be able to jump to a point in time and play, with a given time passing speed, from that point on. 

A prototype has been made in a new Unity project:

![image](https://cloud.githubusercontent.com/assets/5444990/25126749/95bc2f8e-2433-11e7-8e25-cfe5e12f95b2.png)

The prototype provides:
- [x] COBS decoding then protobuf decoding from a file
- [x] replaying the entire simulation in 10 seconds, no matter the length
- [ ] correct replay speed (depends on #153)
- [x] jumping to a point in time
- [ ] loading messages on demand to reduce memory consumption (not needed yet, files fit in memory)
 
Merging the prototype into http://github.com/oliverlee/bikesim is a priority right now.

**Comment by mickvangelderen** [comment]
@oliverlee wrote this down for future reference and sort my head out.
